scripts/install-svc.sh

```
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Define colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Define variables
+SERVICE_NAME="nanodns"
```

Shellcheck

[warning] SERVICE_NAME appears unused. Verify use (or export if used externally).

(SC2034)